### PR TITLE
typescript code generation fixes for rxjs 7

### PR DESCRIPTION
**Pull request by florianraffl**
#3463

### CODE REVIEW

File: src/NSwag.CodeGeneration.TypeScript/Models/TypeScriptFrameworkRxJsModel.cs [hunk 1]
```diff
@@ -8,29 +8,32 @@
```
Comment (RicoSuter) [inline src/NSwag.CodeGeneration.TypeScript/Models/TypeScriptFrameworkRxJsModel.cs:8]: Please use spaces instead of tabs

**Comment by florianraffl** [comment]
@RicoSuter do you know why this CI build check is failing? I think it doesn't have anything to do with my changes or am I wrong?

**Comment by RicoSuter** [comment]
Hmm, building the MSI installer times out in the CI pipeline... no clue how to fix it except excluding it :-)

**Comment by RicoSuter** [comment]
Ok, maybe its fixed with this PR: https://github.com/RicoSuter/NSwag/pull/3493
Can you update your branch with master so we see whether it fixes CI here?

**Comment by florianraffl** [comment]
We had nearly the same problem in our company ... but on an self hosted azure pipelines agent ... there we fixed it by adding the build argument `/p:RunWixToolsOutOfProc=true` to the build step according to:
https://stackoverflow.com/questions/42167960/building-wix-on-visual-studio-team-services-times-out
maybe this also helps you :-)

Ok thanks for the response, I will make a dummy commit to rerun the CI build :-)

**Comment by RicoSuter** [comment]
Ok looks good, ready to merge i suppose?

**Comment by florianraffl** [comment]
Yea that would be perfekt! :-)

**Comment by RicoSuter** [comment]
Thanks for the PR.

**Pull request merged**